topo/node/arista/arista.go

```diff
@@ -366,10 +366,21 @@ func defaults(pb *tpb.Node) *tpb.Node {
 		pb.Labels["vendor"] = tpb.Vendor_ARISTA.String()
```

Comment:
for the services defaults above, I can see two desired behaviors:

1) a user wants to override to a custom gnmi port but still wants all of the other default services to come up so they specify only the gnmi change

2) a user wants a custom gnmi port but nothing else so they only specify gnmi

I think number 2 is less confusing so arista is correct, please make sure the other vendors follow that pattern as well (defaults are only added if services is nil)

Reply:
Agreed, second option seems better, ensured that service defaults are added only if services is nil.